src/web/components/Column/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Column({ children, size }) {
+  let classes = 'column';
```

Comment:
I don't really like `let` variables 😝 

For learning purpose, can you refactor it only using `const` statement (and early returns ? 😛 )

Reply:
And also, I think this is overengineering the component. Flex can't be 0 and people knows that 😛 

I think we should simply return `is-${size}` if `size` is passed, else simply use the default `column` without anything else 😄